axiom/scripts/axiomatic.py

```diff
@@ -121,11 +121,15 @@ def parseOptions(self, args):
             # If a reactor is being selected, it must be done before the store
             # is opened, since that may execute arbitrary application code
             # which may in turn install the default reactor.
-            if "--reactor" in args:
-                reactorIndex = args.index("--reactor")
-                shortName = args[reactorIndex + 1]
-                del args[reactorIndex:reactorIndex + 2]
-                self.opt_reactor(shortName)
+            for index, arg in enumerate(args):
+                if arg.startswith("--reactor"):
+                    if "=" in arg:
+                        shortName = arg.split("=")[1]
```

Comment:
I think Coveralls is wrong; the branch *is* covered, it's just not covered in the same process as coveralls is checking coverage in (because the tests run axiomatic/twistd as a subprocess).